examples/fluids/qfunctions/blasius.h

```diff
@@ -128,35 +109,28 @@ CEED_QFUNCTION(ICsBlasius)(void *ctx, CeedInt Q,
   CeedScalar (*q0)[CEED_Q_VLA] = (CeedScalar(*)[CEED_Q_VLA])out[0];
 
   const BlasiusContext context = (BlasiusContext)ctx;
-  const CeedScalar cv     = context->newtonian_ctx.cv;
-  const CeedScalar cp     = context->newtonian_ctx.cp;
-  const CeedScalar gamma  = cp/cv;
-  const CeedScalar mu     = context->newtonian_ctx.mu;
-
-  const CeedScalar theta0 = context->theta0;
-  const CeedScalar P0     = context->P0;
-  const CeedScalar delta0 = context->delta0;
-  const CeedScalar Uinf   = context->Uinf;
+  const CeedScalar cv         = context->newtonian_ctx.cv;
+  const CeedScalar mu         = context->newtonian_ctx.mu;
+  const CeedScalar theta0     = context->theta0;
+  const CeedScalar P0         = context->P0;
+  const CeedScalar delta0     = context->delta0;
+  const CeedScalar Uinf       = context->Uinf;
   const CeedScalar x_inflow   = context->x_inflow;
-
+  const CeedScalar gamma      = HeatCapacityRatio(&context->newtonian_ctx);
   const CeedScalar e_internal = cv * theta0;
   const CeedScalar rho        = P0 / ((gamma - 1) * e_internal);
-  const CeedScalar x0         = Uinf*rho / (mu*25/ (delta0*delta0) );
-  CeedScalar u, v, t12;
+  const CeedScalar x0         = Uinf*rho / (mu*25/(delta0*delta0));
```

Comment:
It seems like `theta0` needs to be specified somehow compatible with `T_wall` and `T_inf`. I think it probably shouldn't be used at all. I think it should probably be using `T_inf` https://en.wikipedia.org/wiki/Blasius_boundary_layer#Howarth_transformation

Reply:
Oops, I meant to fix this but completely forgot about it. Thanks, dropping `theta0` in a new PR.

Reply:
Cool, sorry about the timing in my review.